### PR TITLE
Undoing political decision

### PR DESCRIPTION
**Pull request by Dum4G**
In the name of my participance in [iptv-org](https://github.com/iptv-org) core member group I'm ressurecting from dead to prevent political driven mess in the playlist I care of. Hope I got syntax right on this one.

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -2822,7 +2822,7 @@ BollywoodHD.ru,Bollywood HD,,,,RU,,,c/RU,rus,movies,FALSE,,,,,https://i.imgur.co
 BollywoodTV.ro,Bollywood TV,,,,RO,,,c/RO,ron,movies,FALSE,,,,http://www.bollywoodtv.ro/,https://i.imgur.com/YLQRnXt.png
 BolNews.pk,Bol News,,,,PK,,,c/PK,urd,news,FALSE,,,,https://www.bolnews.com/,https://i.imgur.com/chHLi5u.png
 BoloNewsHD.pk,Bolo News HD,,,,PK,,,c/PK,eng,news,FALSE,,,,,https://i.imgur.com/8sactJA.png
-Bolt.ua,Bolt,,,Star Media TV;UAMedia Group,UA,,,c/RU;c/UA;c/MD,rus;ukr,movies;series,FALSE,2016-06-10,,,http://bolt.starmediatv.ru/,https://i.imgur.com/YdIHXOR.png
+Bolt.ua,Bolt,,,Star Media TV;UAMedia Group,UA,,,c/RU;c/UAc/MD,rus;ukr,movies;series,FALSE,2016-06-10,,,http://bolt.starmediatv.ru/,https://i.imgur.com/YdIHXOR.png
```
Comment (freearhey) [inline data/channels.csv:2825]:
```suggestion
Bolt.ua,Bolt,,,Star Media TV;UAMedia Group,UA,,,c/RU;c/UA;c/MD,rus;ukr,movies;series,FALSE,2016-06-10,,,http://bolt.starmediatv.ru/,https://i.imgur.com/YdIHXOR.png
```

File: data/channels.csv [hunk 2]
```diff
@@ -12366,7 +12366,7 @@ KingTV.cd,King TV,,,,CD,,,c/CD,fra,religious,FALSE,,,,,https://i.imgur.com/GuL3Y
 KingTV.pk,King TV,,,,PK,,,c/PK,eng,,FALSE,,,,http://www.kingtelevision.tv/,https://i.imgur.com/dujrwRE.png
 KingTV.ug,King TV,,,,UG,,,c/UG,eng,,FALSE,,,,,https://i.imgur.com/CTqUUaK.png
 Kino.si,Kino,,,,SI,,,c/SI,slv,movies,FALSE,,,,https://pro-plus.si/,https://i.imgur.com/XudSdAe.png
-Kino1.ua,Kino 1,,,Surface TV,UA,,Kyiv,c/UA;c/LV,rus,movies,FALSE,2017-01-01,,,https://www.poverkhnost.tv/pages/index.php?c=10&s=155,https://i.imgur.com/FWVxBWI.png
+Kino1.ua,Kino 1,,,Surface TV,UA,,Kyiv,c/UAc/LV,rus,movies,FALSE,2017-01-01,,,https://www.poverkhnost.tv/pages/index.php?c=10&s=155,https://i.imgur.com/FWVxBWI.png
```
Comment (freearhey) [inline data/channels.csv:12369]:
```suggestion
Kino1.ua,Kino 1,,,Surface TV,UA,,Kyiv,c/UA;c/LV,rus,movies,FALSE,2017-01-01,,,https://www.poverkhnost.tv/pages/index.php?c=10&s=155,https://i.imgur.com/FWVxBWI.png
```

File: data/channels.csv [hunk 3]
```diff
@@ -12384,9 +12384,9 @@ Kinomix.ru,Kinomix,Kinomiks;Киномикс,,Gazprom Media,RU,,,c/RU,rus,movies
 KinoNova.bg,Kino Nova,Кино Нова,,,BG,,,c/BG,bul,movies,FALSE,,,,http://kinonova.bg/,https://i.imgur.com/hMqEY0J.png
 Kinopokaz.ru,Kinopokaz,Кинопоказ,,,RU,,,c/RU,rus,movies,FALSE,,,,http://www.kinopokaz.tv/,https://i.imgur.com/nNjMyAq.png
 KinoPolska.pl,Kino Polska,,,SPI International Polska,PL,,,c/PL,pol,movies,FALSE,2003-12-20,,,http://kinopolska.pl/,https://i.imgur.com/lbl8K23.png
-KinoPolskaInternational.pl,Kino Polska International,,,,PL,,,c/UK;c/US;c/UA;c/CA,eng,movies,FALSE,,,,https://spiintl.com/channels/kino-polska-international,https://i.imgur.com/paIaJMP.png
+KinoPolskaInternational.pl,Kino Polska International,,,,PL,,,c/UK;c/US;c/UAc/CA,eng,movies,FALSE,,,,https://spiintl.com/channels/kino-polska-international,https://i.imgur.com/paIaJMP.png
```
Comment (freearhey) [inline data/channels.csv:12387]:
```suggestion
KinoPolskaInternational.pl,Kino Polska International,,,,PL,,,c/UK;c/US;c/UA;c/CA,eng,movies,FALSE,,,,https://spiintl.com/channels/kino-polska-international,https://i.imgur.com/paIaJMP.png
```

File: data/channels.csv [hunk 4]
```diff
@@ -12384,9 +12384,9 @@ Kinomix.ru,Kinomix,Kinomiks;Киномикс,,Gazprom Media,RU,,,c/RU,rus,movies
...
 KinoPolska.pl,Kino Polska,,,SPI International Polska,PL,,,c/PL,pol,movies,FALSE,2003-12-20,,,http://kinopolska.pl/,https://i.imgur.com/lbl8K23.png
-KinoPolskaInternational.pl,Kino Polska International,,,,PL,,,c/UK;c/US;c/UA;c/CA,eng,movies,FALSE,,,,https://spiintl.com/channels/kino-polska-international,https://i.imgur.com/paIaJMP.png
+KinoPolskaInternational.pl,Kino Polska International,,,,PL,,,c/UK;c/US;c/UAc/CA,eng,movies,FALSE,,,,https://spiintl.com/channels/kino-polska-international,https://i.imgur.com/paIaJMP.png
 KinoPolskaMuzyka.pl,Kino Polska Muzyka,,,,PL,,,c/PL,pol,music,FALSE,,,,http://kinopolskamuzyka.pl/,https://i.imgur.com/lE1IlPQ.png
-KinoPolskaMuzykaInternational.pl,Kino Polska Muzyka International,,,,PL,,,c/UK;c/US;c/UA;c/CA,eng,music,FALSE,,,,https://spiintl.com/channels/kino-polska-muzyka-international,https://i.imgur.com/A9akiaf.png
+KinoPolskaMuzykaInternational.pl,Kino Polska Muzyka International,,,,PL,,,c/UK;c/US;c/UAc/CA,eng,music,FALSE,,,,https://spiintl.com/channels/kino-polska-muzyka-international,https://i.imgur.com/A9akiaf.png
```
Comment (freearhey) [inline data/channels.csv:12389]:
```suggestion
KinoPolskaMuzykaInternational.pl,Kino Polska Muzyka International,,,,PL,,,c/UK;c/US;c/UA;c/CA,eng,music,FALSE,,,,https://spiintl.com/channels/kino-polska-muzyka-international,https://i.imgur.com/A9akiaf.png
```

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by Dum4G** [comment]
Fixed the above suggestion but I'm done with Github UI and the way it handles line ending

**Review by freearhey — APPROVED** [review]

**Pull request merged**